File: praktikum/PYTHON/rata.py
def hitung_rata():
    n=int(input("masukkan jumlah bilangan: "))
    total=0
    for i in range(1, n+1):
        x=float(input(f"bilangan ke-{i}: "))
        total=total+ x
    rata=total / n
    print(f"rata-rata = {rata:.2f}")
hitung_rata()

File: praktikum/C++/rata.c
#include <stdio.h>
void hitung_rata(){
 int n;
 printf("masukkan jumlah bilangan: ");
 scanf("%d", &n);

 float total=0, x;
 for(int i=1;i<=n;i++){
  printf("bilangan ke-%d: ", i);
  scanf("%f", &x);
  total=total+ x;
 }

 float rata=total/n;
 printf("rata-rata= %.2f\n", rata);
}

int main(){
 hitung_rata();
 return 0;
}